src/protagonist/API/Features/CustomHeaders/Converters/CustomHeaderConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace API.Features.CustomHeaders.Converters;
+
+public static class CustomHeaderConverter
```

Comment:
this class needs comments

Reply:
Good call - I missed the fact that the other converters have comments. I'll get to work on that 👍